### PR TITLE
Read password from stdin when admin resets password

### LINKED ISSUE #14076: grafana-cli admin password reset should get the password from stdin
### What Grafana version are you using?
Grafana 1.14.6
### What did you do?
Tried to reset the admin password from the grafana-cli.

`grafana-cli admin reset-admin-password myeasypassword`

The admin password is being set on a command line [argument](https://github.com/grafana/grafana/blob/b47a4954c9b0208869298c75e97c212430565f1e/pkg/cmd/grafana-cli/commands/reset_password_command.go#L16). This command might be saved on bash history and it can be a security issue. 

We could instead get the password from stdin.

### PR DESCRIPTION
**Pull request by praveensastry**
Fixes #14076

### CODE REVIEW

**Comment by marefr** [comment]
You've added Grafana as a dependency - 417 files changed?

**Comment by praveensastry** [comment]
> You've added Grafana as a dependency - 417 files changed?

@marefr I updated vendor dependencies using this dep ensure -add golang.org/x/crypto/ssh/terminal and that has resulted in this. Could you please help me understand why this happened?

**Comment by praveensastry** [comment]
@marefr I figured why this is happening, my forked repo is at $GOPATH/src/praveensastry/grafana and with this configuration when i update deps it's adding grafana as a dependency :(

**Comment by marefr** [comment]
Cool

File: pkg/cmd/grafana-cli/commands/reset_password_command.go [hunk 1]
```diff
@@ -2,18 +2,26 @@ package commands
...
 	"github.com/grafana/grafana/pkg/util"
+	"golang.org/x/crypto/ssh/terminal"
 )
 
 const AdminUserId = 1
 
 func resetPasswordCommand(c CommandLine) error {
-	newPassword := c.Args().First()
+	fmt.Print("New password: ")
+	bytePassword, err := terminal.ReadPassword(syscall.Stdin)
```
Comment (marefr) [inline pkg/cmd/grafana-cli/commands/reset_password_command.go]: Will this be backward compatible to `grafana-cli admin reset-admin-password <password>`?

**Review by marefr — CHANGES_REQUESTED** [review]
It works as expected for the password prompt. However it doesn't work to pipe a password to the command like `echo "password" | ./bin/grafana-cli admin reset-admin-password`. Either fix pipe or introduce an option to provide password as an arg in case people have some kind of automation that requires it.

**Comment by bergquist** [comment]
@praveensastry did you have a chance to work on this?

**Comment by marefr** [comment]
@praveensastry Closing this due to lack of response. Just let us know if you would like to continue work on this and we'll reopen this PR. Thanks

**Pull request closed without merging**